packages/react-table/src/components/Table/Table.tsx

```diff
@@ -89,7 +90,7 @@ export interface TableProps extends OUIAProps {
   /** The desired direction to show the dropdown when clicking on the actions Kebab. Can only be used together with `actions` property */
   dropdownDirection?: 'up' | 'down';
   /** The toggle of the actions menu dropdown. A KebabToggle or DropdownToggle component */
-  actionsToggle?: React.ReactElement;
+  actionsToggle?: (props: CustomActionsToggleProps) => React.ReactNode;
```

Comment:
Is this a breaking change?

Reply:
(for others who find this, see https://github.com/patternfly/patternfly-react/issues/6606)